backend/src/labelit/plugins/realtime_video_dimensional/models/realtime_video_dimensional_task.py

```diff
@@ -0,0 +1,46 @@
+from django.db import models
+from labelit.models.task import Task
+from django.core.exceptions import ValidationError
+from labelit.models.annotation import Annotation
+from django.http import Http404
+
+
+class RealtimeVideoDimensionalTask(Task):
+    class Meta:
+        app_label = "labelit"
+
+    def __str__(self):
+        return "<RealtimeVideoDimensionalTask ({}): {}>".format(self.pk, self.name)
+
+    def validate_labels(self, labels, is_final):
+        pass  # TODO
+
+    def _get_stats(
+        self,
+        done_annotations,
+    ):
+        stats = {}
+        # TODO
```

Comment:
An oversight ?

Reply:
No, for now this task won't return stats, it's not needed for the core demo of feature so I left it for later